components/raftstore/src/store/worker/pd.rs

```diff
@@ -523,31 +523,49 @@ where
     }
 }
 
-const HOTSPOT_KEY_RATE_THRESHOLD: u64 = 128;
-const HOTSPOT_QUERY_RATE_THRESHOLD: u64 = 128;
-const HOTSPOT_BYTE_RATE_THRESHOLD: u64 = 8 * 1024;
+const HOTSPOT_READ_KEY_RATE_THRESHOLD: u64 = 128;
+const HOTSPOT_READ_QUERY_RATE_THRESHOLD: u64 = 128;
+const HOTSPOT_READ_BYTE_RATE_THRESHOLD: u64 = 8 * 1024;
+
+const HOTSPOT_WRITE_KEY_RATE_THRESHOLD: u64 = 32;
```

Comment:
we also need a threshold for the write query rate, how about 32?

Reply:
But we don't use it now. Perhaps could add it later

Reply:
Done